templates/events/event.html

```diff
@@ -478,6 +478,23 @@ <h5>Longitude </h5>
             </div>
             {% endif %}
 
+            <!-- displayOnlyWithSuperEvent -->
```

Comment:
Hi @bingzhang, It looks like these are coming from develop branch. Maybe the source branch needs to be created from the master branch. Could you please check?